### ISSUE TITLE
Poetry 1.1.12 does not recognize "plugin" as a valid command

### ISSUE THREAD

**Issue by limsammy** [python-poetry/poetry#4928]
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: Pop!_OS 21.10
- **Poetry version**: 1.1.12
- **pyproject.toml**: [pyproject.toml](https://gist.github.com/limsammy/c75ab2a8f6c886f88873aef2bcb47e6f)

## Issue

Heyo,

Finally beginning to migrate all of our python tools and services to using poetry (absolutely love poetry btw, it is so very elegant! truly encompasses the python zen :smile: )

In short, it seems as though my poetry installation was borked at some point. It does not recognize `plugin` as a command:

```shell
❯ poetry plugin add poetry-dotenv-plugin -vvv
The command "plugin" is not defined.

❯ poetry plugin
The command "plugin" is not defined.

❯ poetry plugin add poetry-dotenv-plugin -vvv
The command "plugin" is not defined.
``` 

I have tried reinstalling, but still does not recognize `plugin` as a command. 

Extra info: I am running poetry within a `virtualenv`. I have installed poetry both globally and within my virtual environments.

**Comment by bmarroquin** [comment]
The plugin system is a 1.2 feature. You can use it with the alpha release. 

https://github.com/python-poetry/poetry/releases/tag/1.2.0a1
